internal/manifests/manifests.go

```diff
@@ -54,13 +49,18 @@ type Manifests struct {
 	db            *gorm.DB
 	log           logrus.FieldLogger
 	objectHandler s3wrapper.API
+	usageAPI      usage.API
 }
 
 func (m *Manifests) CreateClusterManifest(ctx context.Context, params operations.CreateClusterManifestParams) middleware.Responder {
 	manifest, err := m.CreateClusterManifestInternal(ctx, params)
 	if err != nil {
 		return common.GenerateErrorResponder(err)
 	}
+	err = m.setUsage(true, manifest, params.ClusterID)
```

Comment:
a similar call with active=false should be added to `DeleteClusterManifest`

Reply:
@masayag As mentioned in the PR, I added this for easy-later-on-implementation of the delete manifest. But for now we only add usage not removing.

But this is a good point - and we need to discuss how to handle manifest deletions.